<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="cbxStopBits"/>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="cbxFlowControl"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="cbxBaudRate"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>流 .控</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cbxPortName"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="cbxDataBits"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="cbxParity"/>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnOpen">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>串  口</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="btnRefreshPort">
       <property name="text">
        <string>刷新端口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rbtnRcvASCII">
          <property name="text">
           <string>ASCII</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnRcvHEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkAutoLine">
        <property name="text">
         <string>自动换行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chxShowSend">
        <property name="text">
         <string>显示发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chxShowTime">
        <property name="text">
         <string>显示时间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkRepeatSnd">
       <property name="text">
        <string>重复发送</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rbtnSndHEX">
       <property name="text">
        <string>HEX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rbtnSndASCII">
       <property name="text">
        <string>ASCII</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbTimeInterval">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnStopRepeat">
       <property name="text">
        <string>停止发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="rcvTxtEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sndTxtEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>422</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbxSndHistory">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearRcv">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveRcv">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
